src/pages/ArticleDetails.jsx

```diff
@@ -36,49 +52,31 @@ function ArticleDetails() {
       alert("Delete not successfull !!");
     }
   };
+
   return (
     <div className="w-3/5 mx-auto my-10">
       {loader && <Loader />}
       {articleloader || !article ? (
         <div className="flex flex-col gap-5">
-          <div className="rounded-lg h-14 w-full bg-gradient-to-r from-slate-400 to-slate-300 animate-pulse"></div>
-          <div className="h-14 w-full flex justify-between">
-            <div className="flex gap-3">
-              <span className="w-8 h-8 rounded-full bg-gradient-to-r from-slate-400 to-slate-300 animate-pulse"></span>
-              <span className=" w-20 h-7 bg-gradient-to-r from-slate-400 to-slate-300 animate-pulse"></span>
-            </div>
-            <span className="w-12 h-8 bg-gradient-to-r from-slate-400 to-slate-300 animate-pulse"></span>
-          </div>
-          <div className="rounded-lg h-7 w-full bg-gradient-to-r from-slate-400 to-slate-300 animate-pulse"></div>
-          <div className="rounded-lg h-7 w-full flex gap-3">
-            <div className="flex-1 rounded-lg bg-gradient-to-r from-slate-400 to-slate-300 animate-pulse"></div>
-            <div className="flex-1 rounded-lg bg-gradient-to-r from-slate-400 to-slate-300 animate-pulse"></div>
-          </div>
-          <div className="rounded-lg h-7 w-full bg-gradient-to-r from-slate-400 to-slate-300 animate-pulse"></div>
-          <div className="rounded-lg h-7 w-full flex gap-3">
-            <div className="flex-[0.5] rounded-lg bg-gradient-to-r from-slate-400 to-slate-300 animate-pulse"></div>
-            <div className="flex-1 rounded-lg bg-gradient-to-r from-slate-400 to-slate-300 animate-pulse"></div>
-          </div>
-          <div className="rounded-lg h-7 w-full flex gap-3">
-            <div className="flex-[0.7] rounded-lg bg-gradient-to-r from-slate-400 to-slate-300 animate-pulse"></div>
-            <div className="flex-[0.3] rounded-lg bg-gradient-to-r from-slate-400 to-slate-300 animate-pulse"></div>
-          </div>
-          <div className="rounded-lg h-7 w-3/5 bg-gradient-to-r from-slate-400 to-slate-300 animate-pulse"></div>
+          {/* Placeholder skeleton UI */}
```

Comment:
We need to display the skeleton loader UI till the article details are being loaded